package.json

```diff
@@ -5,8 +5,12 @@
   "main": "index.js",
   "scripts": {
     "start": "API_ENDPOINT=http://localhost:8080 webpack serve --port 3000",
+    "start:demo": "serve ./demo -s -p 3001",
     "build": "rm -rf ./dist && NODE_ENV=production API_ENDPOINT=https://api-server-sl9i9.ondigitalocean.app webpack && node ./merge-dist-output.js",
-    "test": "elm-test"
+    "test": "elm-test",
+    "elm": "elm",
+    "format": "elm-format --yes",
+    "prepare": "husky install"
```

Comment:
What does this script do exactly and in what cases would it run?

Reply:
On running `yarn`, the prepare hook gets run which installs the required git hooks for husky to work. This command is not executed by the user.